rules/order/tests/StmtOrderTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Order\Tests;
+
+use PhpParser\Node\Stmt\Class_;
+use PhpParser\Node\Stmt\Property;
+use PhpParser\Node\Stmt\PropertyProperty;
+use Rector\Core\HttpKernel\RectorKernel;
+use Rector\Order\StmtOrder;
+use Symplify\PackageBuilder\Tests\AbstractKernelTestCase;
+
+final class StmtOrderTest extends AbstractKernelTestCase
+{
+    /**
+     * @var string[]
+     */
+    private const DESIRED_STMT_ORDER = ['first', 'second', 'third'];
+
+    /**
+     * @var string[]
+     */
+    private const CURRENT_STMT_ORDER = ['third', 'first', 'second'];
+
+    /**
+     * @var int[]
+     */
+    private const OLD_TO_NEW_KEYS = [
+        0 => 0,
+        1 => 2,
+        2 => 1,
+    ];
+
+    /**
+     * @var StmtOrder
+     */
+    private $stmtOrder;
+
+    protected function setUp(): void
+    {
+        $this->bootKernel(RectorKernel::class);
+
+        $this->stmtOrder = self::$container->get(StmtOrder::class);
+    }
+
+    public function testCreateOldToNewKeys(): void
+    {
+        $actual = $this->stmtOrder->createOldToNewKeys(self::DESIRED_STMT_ORDER, self::CURRENT_STMT_ORDER);
+        $expected = [
+            0 => 1,
+            1 => 2,
+            2 => 0,
+        ];
+        $this->assertSame($expected, $actual);
+    }
```

Comment:
Could you add more test fixture than just 1? 

It will help us in the future, when it breaks.
Like keeping the order untouched and inversed order

Reply:
Btw, what do you think about data providers?